src/sbt-test/lagom-conductr-bundle/lagom-java-service-execution/build.sbt

```diff
@@ -0,0 +1,50 @@
+import org.scalatest.Matchers._
+
+scalaVersion in ThisBuild := "2.11.7"
+version in ThisBuild := "0.1.0-SNAPSHOT"
+
+lazy val `lagom-service-api` = (project in file("lagom-service-api"))
+  .settings(libraryDependencies += lagomJavadslApi)
+
+lazy val `lagom-service-impl` = (project in file("lagom-service-impl"))
+  .enablePlugins(LagomJava)
+  .dependsOn(`lagom-service-api`)
+
+// Test assertions
+val verifyConductInfo = taskKey[Unit]("")
+verifyConductInfo := {
+  val output = conductInfo()
+  output should include("lagom-service-impl")
+}
+
+def conductInfo(): String =
+  (Process("conduct info") #| Process(Seq("awk", "{print $2}"))).!!
+
+InputKey[Unit]("verifyIsStarted") := {
+  val args = Def.spaceDelimited().parsed
+  val bundleName = args(0)
+  DevModeBuild.waitFor[String](
+    bundleStatus(bundleName).trim,
+    _.equals("101"), // 101 -->  #REPlica==1  #STaRting==0  #RUNning== 1
+    (actual:String) => {
+      val message = s"Timeout awaiting [$bundleName] to start."
+      org.scalatest.Matchers.fail(message)
+    }
+  )
+}
+
+def bundleStatus(bundleName:String): String ={
+
+  (Process("conduct info") #| Process(Seq("grep", bundleName))  #| Process(Seq("awk", "{print $3 $4 $5}"))  ).!!
+}
```

Comment:
FYI you can subscribe to SSE events and wait to be notified if a bundle has started. We do this in the CLI. We also have ConductR client lib to make this a bit easier. 

We can leave this though.